playbooks/azure/openshift-cluster/group_vars/all/yum_repos.yml

```diff
@@ -33,13 +33,6 @@ azure_node_repos:
     sslclientkey: /var/lib/yum/client-key.pem
     enabled: yes
 
-  - name: rhel-7-fast-datapath-rpms
```

Comment:
Azure team is developing on master and backporting things to 3.9, I believe.  @kwoodson ptal.

Reply:
Currently we are using the 3.10 branch.  I think we've missed the 3.9 window for release and will be looking to release on 3.10/3.11 in the near future.